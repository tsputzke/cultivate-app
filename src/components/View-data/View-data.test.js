import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme'
import toJson from 'enzyme-to-json'
import ViewData from './View-data'

configure({adapter: new Adapter()});  

describe(`ViewData component`, () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <BrowserRouter>
        <ViewData />
      </BrowserRouter>,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders ViewData by default', () => {
    const wrapper = shallow(<ViewData />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})