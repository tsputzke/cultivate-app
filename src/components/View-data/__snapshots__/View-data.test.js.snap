// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewData component renders ViewData by default 1`] = `
<section
  className="view-data"
>
  <h2
    className="no-data"
  >
    No Data to Display
  </h2>
</section>
`;
